src/main/scala/fastq.scala

```diff
@@ -169,17 +272,17 @@ case object fastq {
         .filterNot(_.isEmpty)
         .grouped(4)
         .map { quartet =>
-          if(quartet(3) == "+") {
 
-            Id.parseFrom( quartet(0) ) flatMap { id =>
-              Sequence.fromStringsPhred33( quartet(1), quartet(3) ) map { FASTQ(id,_) }
-            }
+          if(quartet(3) != "+") None
+          else {
+            for {
+              i <- Id.from( quartet(0) ) // or .parseFrom?
+              s <- Sequence.fromStringsPhred33( quartet(1), quartet(3) )
+            } yield FASTQ(i, s)
```

Comment:
I just changed it to be consistent with the `FASTQ.fromStringsPhred33` above. But also I'm not sure about `parseFrom` there (see https://github.com/ohnosequences/fastarious/pull/13/files#r102502440)